File: 3978-unique-middle-element/3978-unique-middle-element.py
class Solution(object):
    def isMiddleElementUnique(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        n = len(nums)//2
        mid = nums[n]
        for i in range(0,len(nums)):
            if i !=n and mid ==nums[i]:
                return False
        return True

File: 3978-unique-middle-element/3978-unique-middle-element.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var isMiddleElementUnique = function(nums) {
    let n = Math.floor(nums.length/2)
    let mid = nums[n]
    for(let i =0;i<nums.length;i++){
        if(i!==n&&mid===nums[i]){
            return false
        }
        
    }
    return true
};